### ISSUE TITLE
prevent getNextColumn from returning hidden column

### ISSUE THREAD

**Issue by gillez** [olifolkerd/tabulator#1788]
**Describe the bug**
In the documentation, the new function getNextColumn is described as 

> The getNextColumn function returns the Column Component for the next visible column in the table, if there is no next column it will return a value of false."  
http://tabulator.info/docs/4.2/components#component-column

However, it also returns hidden columns.  I haven't tested it but I would guess that getPrevColumn does the same thing.

**Tabulator Info**
- Which version of Tabulator are you using?
v4.2.0

- Post a copy of your construct object if possible so we can see how your table is setup
var table = new Tabulator("#table-test", {
  selectable: "highlight",
  cellClick: function(e, cell) {
    let col = cell.getColumn();
    console.log("cellClick in col", col.getField());
    col = col.getNextColumn();
    console.log("next col", col.getField());    
  },
  columns: [
    {title:"Col1", field:"col1"},
    {title:"Col2", field:"col2", visible: false},
    {title:"Col3", field:"col3"},
  ],
  data: [
    { col1: "a1", col2: "a2", col3: "a3" },
    { col1: "b1", col2: "b2", col3: "b3" },
    { col1: "c1", col2: "c2", col3: "c3" }
  ]
});

**To Reproduce**
1. Create the above table. 
2. Click on a cell in column 1 (e.g. a1)
3. Check the console
Output is:

```
cellClick in col col1
next col col2
```

**Expected behavior**
I expected col2 to be ignored as it is hidden.  Expected out should have been:

```
cellClick in col col1
next col col3
```

**Desktop (please complete the following information):**
IOS, Chrome v64

** Extra info **
I tried to create a codepen with the above code.  However, v4.2.0 isn't yet available on CDN so I wasn't able to do this.

Regards,
Giles

**Comment by olifolkerd** [comment]
Hey @gillez 

Thanks for the feedback, i will have a look into a better way of doing this for the 4.3 release that allows you to choose whether you include hidden columns or not.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @gillez 

I have just pushed a fix for this to the 4.4 branch and will include it in todays release.

Cheers

Oli :)
